openedx/features/qverse_features/registration/signals.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import print_function
+from collections import namedtuple
+
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from openedx.features.qverse_features.registration.models import BulkUserRegistration
+
+UserInfo = namedtuple('UserInfo',
+                      ['username', 'email','first_name', 'last_name',
+                       'other_name', 'mobile_number', 'level',
+                       'programme_and_department']
+                     )
+
+
+@receiver(post_save, sender=BulkUserRegistration)
+def make_users_from_csv_file(sender, instance, created, **kwargs):
```

Comment:
nit `make_user_from_csv_file`, because this will take one user at a time.

Reply:
discussed.

Reply:
it will get a single csv file and then create multiple users.

Reply:
`create_user_from_csv_file`

Reply:
kindly ignore this part. it would be updated and implemented later. It was just written for testing purposes to check either the signal is working or not.